### ISSUE TITLE
Disable Nextcloud web updater

### ISSUE THREAD

**Issue by stephdl** [NethServer/dev#6807]
If sysadmin gains access to NC admin page and runs the update procedure the module might break.

See if possible to disable this NC feature, as updates are handled by NS8

https://docs.nextcloud.com/server/latest/admin_manual/configuration_server/config_sample_php_parameters.html#upgrade-disable-web

    'upgrade.disable-web' => false,

----
Thanks to Dnutan https://community.nethserver.org/t/ns8-nextcloud-updates-question/22416/5?u=davidep

**Comment by stephdl** [comment]
QA

- Install nextcloud on NS8, configure it with a LDAP and a virtualhost
- Connect you with the `admin` user and the default password `Nethesis,1234`
- go to the settings `overview` you should not see the web upgrade proposal
- the ns8-nextcloud target is 27.1.3 and an  upgrade exists to [27.1.5  ](https://github.com/nextcloud/server/tags)

This is true for new installation, the case for upgrade is not true.

**Comment by nrauso** [comment]
test: **VERIFIED**

the `upgrade.disable-web` attribute is set:
```
~]# runagent -m nextcloud1 runagent occ config:system:get upgrade.disable-web
true
```

and no upgrade is proposed:
![nc_ns8](https://github.com/NethServer/dev/assets/16102909/d4376f37-abf5-4a18-a759-8a2cc5f6a5c7)

**Comment by stephdl** [comment]
can we release @DavidePrincipi

**Comment by stephdl** [comment]
released as 1.1.2
